lib/elements/stroke.py

```diff
@@ -203,3 +203,9 @@ def to_patches(self, last_patch):
                 patches.append(patch)
 
         return patches
+
+    def validation_errors(self):
+        # The stroke element has no real error to check for
+        if self is None:
```

Comment:
@kaalleen Have you actually seen this happen?  As far as I understand it this should never be possible...

Reply:
No, I did not. It is very unlikely. But we couldn't use the stroke if we have no validation_errors method. You can change it to whatever you think is best. This bit of code was written out of desperation ;)

Reply:
Ah, sorry about that :)